inventory/group_vars/all.yml

```diff
@@ -56,6 +56,31 @@ db_user: ofn_user
 db_host: localhost
 postgres_encoding: en_US.utf8
 
+#----------------------------------------------------------------------
+# Postgresql variables
+
+# These include the default values for postgres, and some empty (extendable) values.
+# Entries marked custom_* can be defined in secrets.yml files, and will be appended
```

Comment:
we need to update the doc replacing secrets.yml with host_vars.

Reply:
I've updated it so it's more generic. I think this'll need a wiki page anyway, and it'll only be configured by core devs anyway, so we shouldn't worry too much.